apps/meteor/app/importer/server/classes/ImporterSelection.ts

```diff
@@ -1,12 +1,19 @@
-import type { IImporterSelection, IImporterSelectionChannel, IImporterSelectionUser } from '@rocket.chat/core-typings';
+import type {
+	IImporterSelection,
+	IImporterSelectionChannel,
+	IImporterSelectionUser,
+	IImporterSelectionContactOrIdentifier,
+} from '@rocket.chat/core-typings';
 
-export class ImporterSelection implements IImporterSelection {
+export class ImporterSelection<WithData extends boolean = true> implements IImporterSelection<WithData> {
```

Comment:
Since this generic type is used only for the new contacts support, wouldn't it be better to create a specialized class for it and then use the generic, I think this would create confusion especially since we are using a boolean for it. It's not explicit to me when I'm reading the importer process. WDYT?

Reply:
I want to use it for the other data types as well, but that would be a breaking change as this type is used on an endpoint.